cypress/integration/InstancesTest.ts

```diff
@@ -13,27 +17,12 @@ describe("Instances Test", () => {
     });
 
     it("Submit", () => {
-      const newInstanceName: string = "Some new instance";
```

Comment:
@ChughShilpa is a problem to leave here this variable?

Reply:
@sjamboro  As now i created generic function for createInstance and thus passing the instance name as parameter while calling the same function. IMO, there's no need to explicitly assigned it to a constant.

Reply:
@ChughShilpa I have created this variable because I needed its value several times and some occurences stays in code even if you created new function. 

Thank you for your answer.

Reply:
i reverted my changes and keep the code as it is